docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -24,6 +24,32 @@ Most of the methods and constants in this module can only be used or tested in r
 
 **To test manual updates with standalone apps**, you can create a [simulator build](/build-reference/simulators.mdx) or [APK](/build-reference/apk.mdx), or make a release build locally with `npx expo run:ios --configuration Release` and `npx expo run:android --variant release` (you don't need to submit this build to the store to test).
 
+### Using expo-updates with private update server
+Example implementation (of server) and example app using private server can be found in repo [custom-expo-updates-server](https://github.com/expo/custom-expo-updates-server)
+
+There is requirement for every private server to implement this [protocol](https://docs.expo.dev/technical-specs/expo-updates-0/)
+
+### Configuration options
+
+Some build-time configuration options are available to configure the behavior of `expo-updates`. On iOS, these properties are set as keys in `Expo.plist` and on Android as `meta-data` tags in `AndroidManifest.xml` adjacent to the tags added during installation. On Android, you may also define these properties at runtime by passing a `Map` as the second parameter of `UpdatesController.initialize()`, and when provided the values will override any values specified in `AndroidManifest.xml`. On iOS, you may set these properties at runtime by calling `[UpdatesController.sharedInstance setConfiguration:]` at any point _before_ calling `start` or `startAndShowLaunchScreen` and the values in this dictionary will override any values specified in `Expo.plist`.
```

Comment:
```suggestion
Some build-time configuration options are available to configure the behavior of `expo-updates`. 

On Android, these options are set as `meta-data` tags adjacent to the tags added during installation in the **AndroidManifest.xml** file. You can also define these options at runtime by passing a `Map` as the second parameter of `UpdatesController.initialize()`, and the provided values will override any value specified in **AndroidManifest.xml**.

On iOS, these properties are set as keys in **Expo.plist** file. You can also set them at runtime by calling `[UpdatesController.sharedInstance setConfiguration:]` at any point before calling `start` or `startAndShowLaunchScreen`, and the values in this dictionary will override any values specified in **Expo.plist**.
```

I've updated this section to the following guidelines from our writing style guide:
- [Filenames as bold text](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text) such as for **Expo.plist**.
- [Referencing Android before iOS](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web)


The text was a bit hard to read so as per the second guideline I divided into sub-parts for both Android and iOS.
- The phrase ", and when provided the values..." seemed it was missing something. Changed it to " and the provided values will override any ..." for now but let me know or feel free to update if you originally meant something else by it.